### PR TITLE
Add web.dev CSS Courses to CSS References

### PR DESCRIPTION
**Pull request by FADL285**
Add Learn CSS & Learn Responsive Design Courses to CSS References

---

By submitting this pull request, I promise that:

- [x] I have read the [contribution guidelines](https://github.com/micromata/awesome-css-learning/blob/master/contributing.md) thoroughly.
- [x] I ensure my submission follows each and every point.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ This list is mainly about [CSS](https://developer.mozilla.org/docs/Web/CSS) –
 
 ## CSS References
 
+- [Learn CSS - Web.dev](https://web.dev/learn/css/) - An evergreen CSS course and reference to level up your web styling expertise.
```
Comment (mischah) [inline readme.md:25]: Please remove this. I don’t think we need another reference in this awesome list.

File: readme.md [hunk 2]
```diff
@@ -22,6 +22,8 @@ This list is mainly about [CSS](https://developer.mozilla.org/docs/Web/CSS) –
...
+- [Learn Responsive Design](https://web.dev/learn/design/) - A course exploring all aspects of responsive design. Learn how to make sites that look great and work well for everyone.
```
Comment (mischah) [inline readme.md:26]: Please move this to the section »Fundamental concepts«. This is a pretty awesome resource.

**Review by mischah — CHANGES_REQUESTED** [review]
Seems that you didn’t read the [contributing guidelines](https://github.com/micromata/awesome-css-learning/blob/master/contributing.md)

Im happy to add the »Responsive Design« course if you update the PR.

File: readme.md (hunk 2 above)
Comment (FADL285) [reply readme.md]: It was completed
Thank you

**Review by mischah — APPROVED** [review]

**Comment by mischah** [comment]
Thanks for the contribution :octocat: ❤️

**Pull request merged**